third-party/zlib/CMakeLists.txt

```diff
@@ -13,6 +13,21 @@ function(zlibMain)
   endif()
 
   importThirdPartyBinaryLibrary("${name}" "${anchor_file}")
+
+  set(dependencies "")
+
+  definePrebuildProject(${name}
```

Comment:
@Smjert, this is an example of the API I was imagining for keeping the "how to build dependencies" alongside each dependency.